index.js

```diff
@@ -1 +1,16 @@
 'use strict'
+const split = require('split2')
+const { CloudLogging } = require('./lib/cloud-logging')
+
+async function CloudPine (options) {
+  const { logName = 'Cloud_Pine', cloudLoggingOptions } = options
+  const logging = new CloudLogging(logName, cloudLoggingOptions)
+
+  await logging.init()
+
+  const stream = split(logging.parseLine.bind(logging), { autoDestroy: true })
```

Comment:
I am a bit confused here:
```
logging.parseLine.bind(logging),
```
Why do we borrow and assign `logging` to `logging`? 🤔

Reply:
It's just to inject the function without losing the context of the `this`.
If I just borrow the function, the `parseLine` uses the `this` to access a couple of helper functions and the `Logging` instance, all of the previous will be lost if I don't properly hard-bind the `logging` instance to the `bind` function itself